src/control/controlcompressingproxy.cpp

```diff
@@ -0,0 +1,24 @@
+#include "control/controlcompressingproxy.h"
+
+#include "moc_controlcompressingproxy.cpp"
+
+// Compressor
+CompressingProxy::CompressingProxy(QObject* parent)
+        : QObject(parent){};
```

Comment:
Please remove redundant semicolons

Reply:
.. and the closing brance to a new line.

Reply:
You don't need to mention this, because it is already enforced by ClangFormat ;)

Reply:
This one was easy. Done!